kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue

```diff
@@ -95,58 +97,90 @@
 
     mixins: [commonCoreStrings, commonSyncElements],
     setup(props, context) {
-      const {
-        addresses: discoveredAddresses,
-        discoveringPeers,
-        discoveryFailed,
-        discoveredAddressesInitiallyFetched,
-      } = useDynamicAddresses(props);
-      const {
-        addresses: savedAddresses,
-        refreshSavedAddressList,
-        savedAddressesInitiallyFetched,
-      } = useSavedAddresses(props, context);
-      const combinedAddresses = computed(() => {
-        return [...savedAddresses.value, ...discoveredAddresses.value];
-      });
-      const initialFetchingComplete = computed(() => {
-        return savedAddressesInitiallyFetched.value && discoveredAddressesInitiallyFetched.value;
+      const { devices: _devices, isFetching, hasFetched, fetchFailed, forceFetch } = useDevices({
```

Comment:
`useDevices` returns `devices` empty before doFetch fulfills its Promise, and it's not updated anymore (`device` is not reactive). So the screen does not show any device even if the composable gets its value.